github/github.go

```diff
@@ -487,7 +487,7 @@ func (r RateLimits) String() string {
 	return Stringify(r)
 }
 
-// RateLimit is deprecated.  Use RateLimits instead.
+// RateLimit is deprecated.  Deprecated: Use RateLimits instead.
```

Comment:
Maybe I could remove the first sentence altogether? I know `golint` would complain, but it's not very helpful.

https://godoc.org/image/jpeg#Reader does that, it just says "Deprecated: Reader is deprecated.".